.gitignore

```diff
@@ -6,3 +6,4 @@ database.sqlite
 node_modules
 ncp-debug.log
 npm-debug.log
+.idea
```

Comment:
cool! but always add a line break after last line, it can be done automatically by IDE. Reason: http://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline

![image](https://cloud.githubusercontent.com/assets/384223/18526612/e9664700-7ac9-11e6-9fc3-e85bc65225b3.png)